### PR TITLE
Setup all the Entities #40013

### PR DESCRIPTION
**Pull request by Roshan-successive**
- Learned about model setups and seed.
- Model setup with initial seed.
- Set up all entities.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,4 @@
 PORT=9000
 NODE_ENV=dev
-MONGO_URL=mongodb://localhost:27017/express-training
\ No newline at end of file
+MONGO_URL=mongodb://localhost:27017/express-training
+secret_key=qwertyuiopasdfghjklzxcvbnm123456
```
Comment (suraj-bha) [inline .env]: delete this file from PR

**Review by suraj-bha — APPROVED** [review]

File: .env (hunk 1 above)
Comment (suraj-bha) [inline .env]: remove this from PR

**Review by suraj-bha — CHANGES_REQUESTED** [review]
Do the required changes.

File: .env (hunk 1 above)
Comment (Roshan-successive) [reply .env]: Deleted .env file

Comment (Roshan-successive) [reply .env]: Removed .env file from the PR

**Review by suraj-bha — APPROVED** [review]

**Pull request merged**